netbox/core/tests/test_api.py

```diff
@@ -91,3 +98,191 @@ def setUpTestData(cls):
             ),
         )
         DataFile.objects.bulk_create(data_files)
+
+
+class BackgroundTaskTestCase(TestCase):
```

Comment:
Missing `test_background_tasks_list_started()` and `test_background_tasks_list_queued()`?